packages/datadog_flutter_plugin/lib/src/traces/ddtraces_method_channel.dart

```diff
@@ -55,10 +55,12 @@ class DdTracesMethodChannel extends DdTracesPlatform {
 
   @override
   Future<Map<String, String>> getTracePropagationHeaders(DdSpan span) async {
-    var result = await methodChannel.invokeMethod(
-        'getTracePropagationHeaders', {'spanHandle': span.handle});
+    var result = await methodChannel.invokeMethod<Map<Object?, Object?>>(
```

Comment:
Yeah, sorry, for some reason I was thinking the generic was specifying the type of the map... it's the type of the return type. Let's leave it as <Object?, Object?> because I have extra checking lower down.